packages/flutter/test/widgets/image_filter_test.dart

```diff
@@ -29,6 +30,37 @@ void main() {
     );
   });
 
+  testWidgets('Image filter - dilate', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      RepaintBoundary(
+        child: ImageFiltered(
+          imageFilter: ImageFilter.dilate(radiusX: 10.0, radiusY: 10.0),
+          child: const Placeholder(),
+        ),
+      ),
+    );
+    await expectLater(
+      find.byType(ImageFiltered),
+      matchesGoldenFile('image_filter_dilate.png'),
+    );
+  }, skip: kIsWeb); // https://github.com/flutter/flutter/issues/101874
+
+  testWidgets('Image filter - erode', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      RepaintBoundary(
+        child: ImageFiltered(
+          // Do not erode too much, otherwise we will see nothing left.
+          imageFilter: ImageFilter.erode(radiusX: 1.0, radiusY: 1.0),
+          child: const Placeholder(),
```

Comment:
The place holder by default draws a box with diagonals with a stroke width of 2 and doesn't survive the erode process. Try increasing the stroke width on both to a reasonable size so it doesn't erode completely for the golden image.

Reply:
Actually, the dilate filter shows up fine in its golden, it's definitely more than 2 pixels wide, but the erode filter creates a blank image.

Reply:
stroke width modified.